README.md

```diff
@@ -5,9 +5,9 @@ One Who] Perceives the Sounds of the World". (from https://en.wikipedia.org/wiki
 
 ## Requirements
 
-    Node v4.x or higher  (developed on 8.9.1)
+    Node v14.x or higher 
```

Comment:
Might be worth bumping this requirement to at least Node 16, since Node 14 is out of LTS in just a few months and this project doesn't get frequent updates. It might also be worth copying the recommendation for node 18 from the changelog to here, as any new users are more likely to first look in the readme than in the changelog history.